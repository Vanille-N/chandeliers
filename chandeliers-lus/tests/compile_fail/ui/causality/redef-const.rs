//! Duplicate definition of a const.
chandeliers_lus::decl! {
    const X : int = 0;
    const X : float = 0.0;
}

fn main() {}
