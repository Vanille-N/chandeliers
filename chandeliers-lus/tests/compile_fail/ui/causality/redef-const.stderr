error: Attempt to redefine Global `X`, when a prior item already defines it
 --> tests/compile_fail/ui/causality/redef-const.rs:4:5
  |
4 |     const X : float = 0.0;
  |     ^^^^^^^^^^^^^^^^^^^^^
  |
note: Already defined here
 --> tests/compile_fail/ui/causality/redef-const.rs:3:5
  |
3 |     const X : int = 0;
  |     ^^^^^^^^^^^^^^^^^
